crates/ruff_linter/src/rules/flake8_builtins/rules/stdlib_module_shadowing.rs

```diff
@@ -129,7 +133,7 @@ fn get_prefix<'a>(settings: &'a LinterSettings, path: &Path) -> Option<&'a PathB
     prefix
 }
 
-fn is_allowed_module(settings: &LinterSettings, module: &str) -> bool {
+fn is_allowed_module(settings: &LinterSettings, minor_version: u8, module: &str) -> bool {
```

Comment:
I'd prefer to pass around a `PythonVersion` in the `ruff_linter` crate where possible (only converting it to a `u8` representing the minor version when crossing the crate boundary and calling a function from the `ruff_python_stdlib` crate) as it's more strongly typed

```suggestion
fn is_allowed_module(settings: &LinterSettings, version: PythonVersion, module: &str) -> bool {
```